### PR TITLE
[Mellanox] Add a new SKU Mellanox-SN4600C-D112C8

### PR DESCRIPTION
**Pull request by shlomibitton**
- What I did
Add a new SKU Mellanox-SN4600C-D112C8.

- How I did it

Add related files to the device folder:

buffer config templates
pg lookup profile
port_config.ini
sai profile
sensor conf
plugins

- How to verify it

- Description for the changelog

- A picture of a cute animal (not mandatory but encouraged)

### CODE REVIEW

File: device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-D112C8/buffers.json.j2 [hunk 1]
```diff
@@ -0,0 +1 @@
+../ACS-MSN4600C/buffers.json.j2
```
Comment (keboliu) [inline device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-D112C8/buffers.json.j2:1]: Stephen is working on a task to change the MSFT SKU buffer confirmation, you can consult him for the new configuration template.

**Comment by liat-grozovik** [comment]
@shlomibitton where do we stands with this PR? did you get any confirmation from E2E and sync with buffer configuration from Stephen?

**Comment by shlomibitton** [comment]
@liat-grozovik 
Will be ready today.

**Comment by liat-grozovik** [comment]
🙏🏻

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by shlomibitton** [comment]
@liat-grozovik 
done.

**Comment by liat-grozovik** [comment]
@stephenxs  and @keboliu  do we need to wait for another PR or can we push it upstream already?

**Pull request closed without merging**